solr/solr-ref-guide/src/parallel-sql-interface.adoc

```diff
@@ -66,6 +66,14 @@ The `/sql` handler is the front end of the Parallel SQL interface. All SQL queri
 
 By default, the `/sql` request handler is configured as an implicit handler, meaning that it is always enabled in every Solr installation and no further configuration is required.
 
+==== Authorization for SQL Requests
+
+If your Solr cluster is configured to use the <<rule-based-authorization-plugin.adoc#,Rule-based Authorization Plugin>>,
+then you need to grant `GET` and `POST` permission on the `/sql`, `/select`, and `/export` endpoints for all collections you intend to execute SQL queries against.
+
+Prior to Solr 9.0, you'll also need to grant `GET` permission on the `/admin/luke` endpoint for all collections queried using SQL; the JDBC driver uses this endpoint to get schema metadata for the collection.
+As of Solr 9.0, the Solr executes the request to the `/admin/luke` endpoint using the internal PKI principal.
```

Comment:
Sorry for delay, I went out of town for a few days.

So there's nothing specifically wrong with this as it is, although it means eventually someone will need to fix the page to remove the "prior to 9.0", likely in like a year+ when we're working on 10.0.

What I would do myself, though, if I was doing it, is add this text here to `branch_8x` since it's only relevant to 8.x users. Then upgrade notes for 9.0 (`main` branch) would explain the change ("Prior to 9.0...As of 9.0..."). This page in 9.0 would just have your 2nd sentence explaining how it works in 9.0, if you thought it was an important detail to mention.

I approach it this way because having versioned docs means you don't have to try to make each version a historical document of what happened in earlier versions, and I'm rather ruthless about avoiding that. But I know others have different opinions about how far we should take it considering our redirect-to-latest strategy, how Google ranks, lack of a version picker, if people even read the upgrade notes, etc., so that's why I say it's fine to leave it the way you have it if you prefer to.

Reply:
Your approach sounds good to me @ctargett ... I'll make some changes to branch_8x! Thanks.